packages/core/src/utils/formatRuneName.test.ts

```diff
@@ -0,0 +1,47 @@
+import { describe, expect, it } from "vitest";
```

Comment:
In general I'm not sure if the function works correctly according to the specification.  The problem here that the spec doesnt exist. Look through the code to figure out what is the current implementation for rune name validation. https://github.com/ordinals/ord/

There are the docs here also for the protocol https://docs.ordinals.com/runes.html but I'm not sure if they are up to date